src/pageObject.js

```diff
@@ -2,15 +1,41 @@
+"use strict";
 
 class Header {
-
-};
+  get logo() {
+    return "Conduit logo";
+  }
+}
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
-};
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
+}
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
+    this.slug = slug;
+    this.url += slug;
+  }
 
-};
+  get commentButton() {
+    return "[Publish comment] button";
+  }
+
+  clickOnCommentButton() {
+    return `Click on the ${this.commentButton}`;
```

Comment:
Similar to the `clickOnLogo` method, the `clickOnCommentButton` method currently returns a string. Consider adding a comment to clarify if this is a placeholder for future implementation.